tests/rules/pds3/test_corss_8xxx.py

```
@@ -0,0 +1,258 @@
+        for pdsf in product_pdsfiles:
+            opus_id_abspaths.add(pdsf.abspath)
+
+        for pdsf in product_pdsfiles:
+            # Every version is in the product set
+            for version_pdsf in pdsf.all_versions().values():
+                assert version_pdsf.abspath in opus_id_abspaths
+
+            # Every viewset is in the product set
+            for viewset in pdsf.all_viewsets.values():
+                for viewable in viewset.viewables:
+                    if 'diagrams/' in viewable.abspath: continue    # skip diagrams
```

Ruff (0.15.21)

[error] Multiple statements on one line (colon)

(E701)